test/media/streaming_engine_integration.js

```diff
@@ -522,6 +522,7 @@ describe('StreamingEngine', function() {
    * Initializes or switches to the given period.
```

Comment:
This comment is out of date.  Can you please update to "Choose streams for the given period"?  Same in player_unit and streaming_engine_unit.